CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.59.0      2024-10-01
+
+* Use new WebSocket-based API for snapshot submissions
+  - Long-lived WebSocket connections have lower overhead for individual snapshots
+    that send statistics data to pganalyze, and avoid repeated HTTP connections
+  - In case of errors when connecting (e.g. due to a proxy, or missing server side support),
```

Comment:
I might word this differently, for example:
 >(e.g. due to a misconfigured proxy, or enterprise server release not including WebSocket support)

Reply:
Yep, good point - revised it a bit.